### PR TITLE
Exposing activation function

### LINKED ISSUE #636: Activation functions not exposed in all models.
Some models have the activation function as parameters, but most don't. 

This needs to be resolved for a study I am conducting.

### PR DESCRIPTION
**Pull request by bruAristimunha**
closes #636

### CODE REVIEW

File: braindecode/models/atcnet.py [hunk 1]
```diff
@@ -119,7 +119,7 @@ def __init__(
         tcn_kernel_size=4,
         tcn_n_filters=32,
         tcn_dropout=0.3,
-        tcn_activation=nn.ELU(),
+        tcn_activation=nn.ELU,
```
Comment (bruAristimunha) [inline braindecode/models/atcnet.py:122]:
```suggestion
        tcn_activation=nn.ELU(),
```

File: braindecode/models/atcnet.py [hunk 2]
```diff
@@ -166,7 +166,7 @@ def __init__(
         self.tcn_kernel_size = tcn_kernel_size
         self.tcn_n_filters = tcn_n_filters
         self.tcn_dropout = tcn_dropout
-        self.tcn_activation = tcn_activation
+        self.tcn_activation = tcn_activation()
```
Comment (bruAristimunha) [inline braindecode/models/atcnet.py:169]:
```suggestion
        self.tcn_activation = tcn_activation
```

File: braindecode/models/biot.py [hunk 3]
```diff
@@ -69,10 +69,10 @@ class _ClassificationHead(nn.Sequential):
         (batch, n_outputs)
     """
 
-    def __init__(self, emb_size: int, n_outputs: int):
+    def __init__(self, emb_size: int, n_outputs: int, activation: nn.Module = nn.ELU):
```
Comment (agramfort) [inline braindecode/models/biot.py:72]: Docstring needs update. Can you check all docstrings?

Comment (bruAristimunha) [reply braindecode/models/biot.py:72]: yes @agramfort!

**Comment by bruAristimunha** [comment]
I thought better of it @agramfort, and I think you're pretty busy this week, I'll try to move a little faster, but your review is always welcome on any PR you want to review 🙏🏽 
image

**Pull request merged**